code/modules/projectiles/projectile/special.dm

```diff
@@ -11,8 +11,10 @@
 	empulse(A, pulse_range, pulse_range)
 	return 1
 
+/obj/item/projectile/ion/blast
+	name = "ion blast"
```

Comment:
I dont see the point of having two projectiles that do exactly the same thing but are named different.
It just adds inconsistency, especially since other weapons still use the "ion bolt" with a aoe.